.coderabbit.yaml

```
@@ -0,0 +1 @@
+# wip
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)